TESTING-INSTRUCTIONS.md

```diff
@@ -268,12 +269,10 @@ In case the report shows "no data", please reimport historical data by following
 
 -   Create a brand new site.
 -   Install a plugin to log every sent email (you can use [WP mail logging](https://wordpress.org/plugins/wp-mail-logging/)).
--   Install and active [this gist](https://gist.github.com/octaedro/864315edaf9c6a2a6de71d297be1ed88) to create an email note. Just download the file and install it as a plugin.
--   After activating the plugin, press `Add Email Notes` to create a note.
+-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications).
```

Comment:
```suggestion
-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases).
```

I'm thinking a link to download the latest release is a shorter path. There are a couple more links like this below, if you agree to change search and replace would be quicker

Reply:
Good catch!